### PR TITLE
Fixes #24686 - Remove border around console to fix cursor offset

### PR DESCRIPTION
**Pull request by UXabre**

### CODE REVIEW

**Comment by UXabre** [comment]
For the reviewers, the issue I'm trying to solve here, is one in which there seems to be a small cursor offset due to rounder border around the console. When there rounded borders + padding is enabled:
image
When it is disabled it's correct:
image

An IMHO nice by-product, is that I manage to spruce up the console UI a little bit, rendering it more in line with the rest of the UI.

Before changes:
image
After changes:
image

**Comment by lizagilman** [comment]
Thanks @UXabre ,
@terezanovotna , how do you feel about the console position?

**Comment by terezanovotna** [comment]
Great improvement with fixing the cursor offset!
Console being in the middle LGTM!

**Comment by UXabre** [comment]
Allright, great to hear that this look good (apparently I'm getting old as I had to look up what LGTM means :p ). One thing that might not be clear is that I currently removed the "hidden password" from the UI for VNC connections. Not sure if this is a wanted feature and if it should stay? Perhaps it's safest to only show it IF there is a password? (and thus not breaking any feature)

File: app/views/hosts/console/vnc.html.erb [hunk 1]
```diff
@@ -13,11 +13,9 @@
           {:id => "troubleshootingButton", :class => "btn btn-default"}
         )
  ) %>
-<p class="toggle-hidden-value"><%= _('Password:') %> <a href="#" class="toggle-hidden-value"><%=icon_text("plus")%></a><span><%= @console[:password] %></span></p>
```
Comment (tbrisker) [inline app/views/hosts/console/vnc.html.erb]: Is this removal on purpose? doesn't seem related to the layout change

**Review by tbrisker — COMMENTED** [review]
One comment inline. I also invited you to the Foreman github organization, if you join and make your membership public your future PRs will automatically trigger CI. for now:
ok to test

**Comment by tbrisker** [comment]
ok to test

File: app/views/hosts/console/vnc.html.erb (hunk 1 above)
Comment (UXabre) [reply app/views/hosts/console/vnc.html.erb]: It was done on purpose, but after reading a bit on the PR that added this, I think the best course of action is to only show it if there ia a VNC password available. I will change this PR to reflect this.
The reason why I removed it, was that I saw "Password: " but the password, in my case, is always empty... which was short-sighted  from my side

Comment (tbrisker) [reply app/views/hosts/console/vnc.html.erb]: I suggest separating that change to a different pr in that case :)

Comment (UXabre) [reply app/views/hosts/console/vnc.html.erb]: Okay, makes sense 👍 I'll remove this change. Thanks for the input

**Comment by UXabre** [comment]
@tbrisker the proposed change has been made and pushed :-)

**Review by tbrisker — APPROVED** [review]
Thanks @UXabre, @lizagilman and @terezanovotna !
